Entity/ContentType.php

```diff
@@ -172,13 +172,6 @@ class ContentType extends JsonDeserializer implements \JsonSerializable
      */
     protected $colorField;
 
-    /**
-     * @var string
-     *
-     * @ORM\Column(name="parentField", type="string", length=100, nullable=true)
-     */
-    protected $parentField;
```

Comment:
Does this not breaking the relation between ContentType and FieldType? No Doctrine warning?

Reply:
Will check it again ...